css/style.css

```diff
@@ -392,6 +392,7 @@ form a {
 
 form input, form button {
   flex: 1;
+  margin: 5px;
```

Comment:
We can remove this, the grid handles spacing

Reply:
You could add margin-bottom: 1em to #content to separate the form and the notice.